### DISCUSSION TITLE
Would it be possible to add text-color and text-background-color extensions?

### DISCUSSION THREAD

**Discussion by zhaoyao91** [remirror/remirror#222] [Ideas]
## Description

How about add extensions to change text color and text background color?

## Possible Implementation

They may be mark extensins. use span with style to wrap the styled text. 

## Checklist

- [ ] I have read the [**contributing**](https://github.com/remirror/remirror/blob/canary/docs/contributing.md) document.

**Comment by ifiokjr** [comment]
That would be great! Would you be comfortable creating a PR?

I can provide guidance.

**Comment by zhaoyao91** [comment]
Yeah, these days I am researching how to use remirror to create an editor suit for the requirements of our project. It's hard, since lacking of deep docs.

I thought wypiwys-editor may be the start point, but the doc only wrote one line to add it, no more explanation about how to customized it: customize the style of menu? remove/add some item? how to write an extension? how to bind extension to menu? is there any concerns/caveat when customizing? ...

I have to dig the source code since then, and I have no confidence to control remirror currently. But once I have made such extensions, I'd glad to PR them.

**Comment by ifiokjr** [comment]
Just letting you know that this is something I really want to see available for `remirror`. I'll start on it next week.
